apps/api/src/server/services/AuthService.ts

```diff
@@ -65,84 +65,68 @@ export class AuthController {
   }
 }
 
-export class AuthService extends BaseService {
+export class AuthService {
```

Comment:
Is `BaseService` being deprecated? If so the import above (line 8) can be removed, and the class itself removed from `Service.ts`.